tests/LibraryTest.php

```diff
@@ -71,4 +71,40 @@ public function testNoHEntryMarkupWithTarget()
         $this->assertArrayNotHasKey('html', $data);
     }
 
+    public function testHandlesHCardWithoutURLProperty()
+    {
+        $url = 'http://example.com/';
+        $html = '<p class="h-card">The Mythical URLless Person</p>';
+        $xray = new p3k\XRay();
+        $data = $xray->parse($url, $html);
+        $this->assertEquals('card', $data['data']['type']);
```

Comment:
This should also test to make sure the `url` property is returned and is the empty value.